### DISCUSSION TITLE
MELHORIA: facilitar o processo de cadastro dos Quick Commands

### DISCUSSION THREAD

**Discussion by danilotavares** [stack-spot/stackspot-roadmap#92] [Stackspot AI]
MELHORIA: facilitar o processo de cadastro de um Quick Command, removendo a necessidade de incluir a tag do comando e o sufixo “.answer” na descrição do resultado.
![anexo](https://github.com/stack-spot/stackspot-roadmap/assets/4457275/356f2146-3a2a-4566-b37e-ca46279603fa)

**Comment by magnolemoszup** [comment]
Fala @danilotavares , blz?
Sua sugestão foi atendida e já está em prod rsrsr. Depois confere lá por favor e da seu feedback?

**Reply by danilotavares** [reply]
maravilha mano [magnolemoszup](https://github.com/magnolemoszup), obrigado =)
